storage/client_range_gc_test.go

```diff
@@ -41,6 +41,20 @@ func TestRangeGCQueueDropReplica(t *testing.T) {
 	// Increment the clock's timestamp to expire the leader lease.
 	mtc.manualClock.Increment(int64(storage.DefaultLeaderLeaseDuration) + 1)
 
+	// Make sure the range is not yet removed from the store.
```

Comment:
Hmm, in order to call `shouldQueue`, we need to expose the range GC queue and the method since the test is in the `storage_test` package.

We can try moving this test to the `storage` package, but `storage_test.multiTestContext` has a convenient testing functionality, I think. For instance, we can add a test for garbage-collecting merged ranges easily using `multiTestContext`. I think writing such test isn't easy if we use `createTestStore` to create a store since `testSender` doesn't support `EndTransactionRequest` right now.